### ISSUE TITLE
Middleware never executed (Bug)

### ISSUE THREAD

**Issue by eightyfive** [Folkloreatelier/laravel-hypernova#2]
Hi,

There is bug in the middleware when choosing to modify the response or not:
https://github.com/Folkloreatelier/laravel-hypernova/blob/master/src/Folklore/Hypernova/Hypernova.php#L124

This line:
```php
$response->headers->has('Content-Type') === 'text/html'
```

Should be (or the like):
```php
strpos($response->headers->get('Content-Type'), 'text/html') !== false
```

Also, as a side note, I believe deciding whether or not to modify the response should be handled in the middleware itself:
```php
// Move conditions in Middleware
        if ($response instanceof BaseResponse &&
            !$response->isRedirection() &&
            (
                !$response->headers->has('Content-Type') ||
                strpos($response->headers->get('Content-Type'), 'text/html') !== false
            )
        ) {
...
```

**Comment by dmongeau** [comment]
Makes a lot of sense, will commit this soon

**Comment by dmongeau** [comment]
Should be fixed in v0.1.1
